### PR TITLE
#163512168 Fix password reset issues

### PR DESCRIPTION
**Pull request by okoroemeka**
#### What does this PR do?
Fix password reset issues

#### Description of Task to be completed?
Have password reset functionality working

#### How should this be manually tested?
clone the repo, cd into it then `run npm install`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories? (If applicable)
[163512168](https://www.pivotaltracker.com/story/show/163512168)

#### Screenshots (If applicable)
N/A

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -40,3 +40,5 @@ SUPER_ADMIN_STATUS=superAdmin
 SUPER_ADMIN_EMAIL=
 SUPER_ADMIN_PASSWORD=
 SUPER_ADMIN_USERNAME=
+
+FRONT_END_DEV_URL=
```
Comment (akhilome) [inline .env.sample]: New line needed here.

**Review by akhilome — COMMENTED** [review]
Hey, @okoroemeka, nice work. Love the fact that you added the new env variable to the `.env.sample` file.

Little nitpick below, please go over it.

Also, please make your PR title more descriptive. "Fix Bug" is very vague. Which bug?

**Review by akhilome — APPROVED** [review]

**Comment by omokaro-faith** [comment]
@okoroemeka Please fix hounds issue if they aren't trivial.

**Comment by okoroemeka** [comment]
> @okoroemeka Please fix hounds issue if they aren't trivial.

The hound issues are not displaying on my end, that's why I have not been able to fix it.

**Review by frostyblok — APPROVED** [review]

**Pull request merged**